app/lib/frontend/templates.dart

```diff
@@ -53,27 +53,36 @@ class TemplateService {
 
     final stableVersionRows = [];
     final devVersionRows = [];
+    PackageVersion latestDevVersion;
     for (int i = 0; i < versions.length; i++) {
       final PackageVersion version = versions[i];
       final String url = versionDownloadUrls[i].toString();
       final rowHtml = _renderVersionTableRow(version, url);
       if (version.semanticVersion.isPreRelease) {
+        latestDevVersion ??= version;
```

Comment:
I'm assuming the versions are sorted with latest first, right?

Reply:
yes, they are.